### PR TITLE
Feature/review model migration

### PR DESCRIPTION
**Pull request by Vkolya**

### CODE REVIEW

File: app/Entities/Review/Review.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Hedonist\Entities\Review;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Review extends Model
+{
+    /**
```
Comment (nemoipaha) [inline app/Entities/Review/Review.php]: Please remove all dockblocks )

File: database/migrations/2018_08_03_103854_create_reviews_table.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reviews', function (Blueprint $table) {
+            $table->increments('id');
+            $table->text('description');
+            $table->unsignedInteger('user_id');
+            $table->unsignedInteger('place_id');
```
Comment (lenchvolodymyr) [inline database/migrations/2018_08_03_103854_create_reviews_table.php]: Make a foreign key on `place_id`, it doesn't matter that a table doesn't exist yet

File: app/Entities/Review/Review.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Hedonist\Entities\Review;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Review extends Model
+{
+
```
Comment (nemoipaha) [inline app/Entities/Review/Review.php]: @Vkolya Empty line.

File: app/Entities/Review/ReviewPhoto.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Hedonist\Entities\Review;
+
+use Illuminate\Database\Eloquent\Model;
+use Hedonist\Entities\Review\Review;
+
+class ReviewPhoto extends Model
+{
+
```
Comment (nemoipaha) [inline app/Entities/Review/ReviewPhoto.php]: Empty line )

**Pull request merged**